### ISSUE TITLE
pretrain script for cifar-10

### ISSUE THREAD

**Issue by Xiaohui9607** [zhangq327/U-MAE#1]
Hi, 

Is it possible that you can share the training script of cifar-10?

Thanks!

**Comment by zhangq327** [comment]
As the official code of MAE has no training script of CIFAR-10, so we use another code repository when training on CIFAR-10 (https://github.com/IcarusWizard/MAE). The implementation of our U-MAE loss has no difference and we follow the default settings of that repository.
